README.md

```diff
@@ -70,12 +78,12 @@ very latest API version. As of this writing, the site is compatible with API ver
       nodejs         <version> (set by ~/dotcom/.tool-versions)
       ruby           <version> (set by ~/dotcom/.tool-versions)
      ```
-     
+
      If you are missing any versions, you should re-run `asdf install`. Related [Github issue about asdf-erlang](https://github.com/asdf-vm/asdf-erlang/issues/57)
 
 1. Install chromedriver (for Elixir acceptance tests using Wallaby)
     ```
-    brew tap caskroom/cask
+    brew cask install google-chrome # if you don't have Chrome already
```

Comment:
Can Chrome be acquired via browser download/installer or does it need to be explicitly added via brew?

Reply:
It just needs to be installed, via any means. Perhaps we can safely leave out this instruction, I thought it might be good to specify somewhere that `chromedriver` needs Chrome to be installed manually since it doesn't list it as a dependency in Homebrew.

Reply:
It just needs to be installed, via any means. Perhaps we can safely leave out this instruction, I thought it might be good to specify somewhere that `chromedriver` needs Chrome to be installed manually since it doesn't list it as a dependency in Homebrew.

Reply:
Ok. I would move it as a note prior to the command.

Reply:
I thought this was required to get Chrome to work headlessly... but may be mistaken

Reply:
There shouldn't be any difference between installation methods; `brew cask` essentially automates the downloading and installing of the same file you'd get from the Chrome web site.

Will change this to a note!